beetsplug/acousticbrainz.py

```diff
@@ -41,66 +46,118 @@ def func(lib, opts, args):
         cmd.func = func
         return [cmd]
 
+    def import_task_files(self, session, task):
+        """Function is called upon beet import.
+        """
+
+        items = task.imported_items()
+        fetch_info(self._log, items)
+
 
 def fetch_info(log, items):
-    """Currently outputs MBID and corresponding request status code
+    """Currently outputs MBID and corresponding request status code.
     """
+    def get_value(map_path):
```

Comment:
The map path could just be the arguments, by adding `*` in front of `map_path`. Then it would be called like `get_value("highlevel", "gender", "value")` without the extra square brackets.